geojson_pydantic/strategies/bbox.py

```diff
@@ -0,0 +1,30 @@
+"""Hypothesis strategies for generating bounding boxes."""
```

Comment:
what d you think about put this code in `geometries.py` ?

Reply:
Everything was all in one mega file but I split this off while trying to make it more organized. I was thinking it's own place mostly because it's defined in `types.py` and not the main geometries. I'm fine whatever way makes the most sense to others though.